src/main/java/com/rakovets/course/java/core/practice/arrays/Task03.java

```diff
@@ -35,7 +37,17 @@ public static void main(String[] args) {
     static double[] getAverageMarks(int[][] marks) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        double[] averageMarks = new double[marks.length];
+        int sumMarks = 0;
+        for (int i = 0; i < marks.length; i++){
+            for (int j = 0; j < marks[0].length; j++) {
+                    sumMarks += marks[i][j];
+                }
+            double averageMark = (double) sumMarks / marks.length;
+            averageMarks[i] = NumberUtil.roundValueToTwoDigitsForMantissa(averageMark);
+            sumMarks = 0;
+            }
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"